src/app/select-trajectory/trajectory-selector/trajectory-selector.component.ts

```diff
@@ -20,10 +20,27 @@ export class TrajectorySelectorComponent implements OnInit, OnDestroy {
       .getAllMeta()
       .subscribe((ts) => {
         this.loading = false
-        this.trajectories = ts
+        this.trajectories = ts.sort((a, b) => {
+          return (
+            0 - (this.getSortingIndex(a) > this.getSortingIndex(b) ? -1 : 1)
```

Comment:
Looks good! Two questions:

1. Why `0 - (this.getSortingIndex(a) > this.getSortingIndex(b) ? -1 : 1)` 
instead of `this.getSortingIndex(a) > this.getSortingIndex(b) ? 1 : -1`?

2. Possibly negligible, but still: This always returns 1 or -1, therefore it _always_ reorders the trajectories, even if not necessary. Might not be relevant for now, but when we have more than the current few trajectories in the future, this could lead to odd sorting-behaviour.

Reply:
Good point, that doesn't make much sense! Fixed here: https://github.com/sitcomlab/simport-learning-app/pull/36/commits/03f6de37c9edbd8b7c61bde9cfb7bf59760c73c4